apps/opik-frontend/src/components/pages/ProjectsPage/ProjectsActionsPanel.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback, useRef, useState } from "react";
+import { Trash } from "lucide-react";
+
+import { Button } from "@/components/ui/button";
+import { Project } from "@/types/projects";
+import useProjectBatchDeleteMutation from "@/api/projects/useProjectBatchDeleteMutation";
+import ConfirmDialog from "@/components/shared/ConfirmDialog/ConfirmDialog";
+import TooltipWrapper from "@/components/shared/TooltipWrapper/TooltipWrapper";
+
+type ProjectsActionsPanelsProps = {
+  projects: Project[];
+};
+
+const ProjectsActionsPanel: React.FunctionComponent<
+  ProjectsActionsPanelsProps
+> = ({ projects }) => {
+  const resetKeyRef = useRef(0);
+  const [open, setOpen] = useState<boolean>(false);
+  const disabled = !projects?.length;
+
+  const { mutate } = useProjectBatchDeleteMutation();
+
+  const deleteProjectsHandler = useCallback(() => {
+    mutate({
+      ids: projects.map((p) => p.id),
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [projects]);
```

Comment:
you could just add `mutate` here. it's not as heavy as the entire `useProjectBatchDeleteMutation()`

Reply:
Fixed